lib/ci/tacos-apply

```diff
@@ -15,6 +15,32 @@ EOF
     exit 1
 fi
 
+if "$GITHUB_PR_DRAFT"; then
+    : no applying a draft! bail.
+    gha-step-summary >/dev/null <<EOF
+### TACOS Apply: $TF_ROOT_MODULE
+
+Applies are not allowed for draft PRs.
+Please set the PR ready for review first.
+
+Apply request refused: PR is a draft
```

Comment:
I tried to touch up the wording. Feel free to just "ack".


```
Draft PRs cannot be applied.
Please try again after code review.

Apply request refused: PR is not ready
```

Reply:
used this wording